src/generic/PermGroups.jl

```diff
@@ -332,18 +363,18 @@ function (G::PermGroup)()
 end
 
 function (G::PermGroup)(a::Array{Int, 1}; checked=true)
```

Comment:
While you have this PR open, can you change the signature to `(a::Array{Int, 1}, checked::Bool = true)`? Keyword arguments have a rather big performance penalty in `julia` (hopefully this will change in the future, see https://github.com/JuliaLang/julia/issues/9551).